src/Uno.UWP/Generated/3.0.0.0/Windows.Devices.Sensors/BarometerReadingChangedEventArgs.cs

```diff
@@ -2,12 +2,12 @@
 #pragma warning disable 114 // new keyword hiding
 namespace Windows.Devices.Sensors
 {
-	#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __MACOS__
+#if false || false || false || false || false
```

Comment:
💅 minor nitpick, that doesn't require a change. This entire file could have been reduced to

```
#pragma warning disable 108 // new keyword hiding
#pragma warning disable 114 // new keyword hiding
namespace Windows.Devices.Sensors
{
	public partial class BarometerReadingChangedEventArgs 
	{

	}
}
```

and it will be automatically done next time we re-run the UWPSyncTool. This really isn't documented anywhere and I've taken an action to 🖊 something up to make it more clear. 👍